server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/mail/Email.scala

```diff
@@ -61,7 +62,18 @@ object Email {
   }
 
   def from(message: (MessageId, Seq[MessageResult])): Email =
-    Email(message._1, sanitizeSize(message._2.head.getSize))
+    Email(message._1, asEmailHeaders(message._2.head.getHeaders.headers()), sanitizeSize(message._2.head.getSize))
+
+  def asEmailHeaders(headers: java.util.Iterator[Header]): Seq[EmailHeader] = {
+    headers.asScala
+      .map(header => EmailHeader(header.getName, header.getValue))
```

Comment:
Let's be crazyyyy and replace `header.getValue` by `header.getRaw` (& of course some formatting wizardry to display the byteSequence as a string) in order to be spec compliant! :-)